src/main/java/jenkins/plugins/gerrit/workflow/GerritReviewStep.java

```diff
@@ -77,15 +98,26 @@ protected Void run() throws Exception {
     }
   }
 
+  @Deprecated
```

Comment:
I would keep it for make the simplest use-case less error prone for people though.
Possibly not your use-case at all, but I believe (I may be wrong of course) that most of the people would only use the Verified label with Jenkins, as they currently do as well for the Gerrit Trigger plugin.